pkgs/development/python-modules/eventlet/default.nix

```diff
@@ -33,17 +33,17 @@ buildPythonPackage rec {
     echo "nameserver 127.0.0.1" > resolv.conf
     export NIX_REDIRECTS=/etc/protocols=${iana-etc}/etc/protocols:/etc/resolv.conf=$(realpath resolv.conf)
     export LD_PRELOAD=${libredirect}/lib/libredirect.so
-
     export EVENTLET_IMPORT_VERSION_ONLY=0
   '';
 
   checkPhase = ''
     runHook preCheck
-
     # test_fork-after_monkey_patch fails on aarch64 on hydra only
     #   AssertionError: Expected single line "pass" in stdout
-    nosetests --exclude test_getaddrinfo --exclude test_hosts_no_network --exclude test_fork_after_monkey_patch
-
+    # test_dns_methods_are_green fails on x86_64
+    #   AssertionError: Expected single line "pass" in stdout
+    nosetests --exclude test_getaddrinfo --exclude test_hosts_no_network --exclude test_fork_after_monkey_patch \
+    --exclude test_dns_methods_are_green
```

Comment:
I am the maintainer of eventlet and it builds on hydra. Why is this change necessary?

Reply:
It didn't build on my two x86_64 systems w/o excluding this test. 
I started working on the old PR, which was based on an old revision of the master branch of nixpkgs. Something must have changed, because now it isn't necessary anymore. 
I reverted the change.